packages/nimble-components/package.json

```diff
@@ -57,8 +57,9 @@
     "@babel/cli": "^7.13.16",
     "@babel/core": "^7.14.6",
     "@babel/preset-env": "^7.13.15",
-    "@ni/eslint-config-javascript": "^3.1.0",
-    "@ni/eslint-config-typescript": "^3.0.5",
+    "@mdx-js/react": "*",
```

Comment:
Can you give more context on the `@mdx-js/react` `devDependency`? What's it needed for and why use a `*` version?

Reply:
It's required by the nimble-components build, but after regenerating the package-lock, it doesn't always get installed. Rather than remove this declaration and depend solely on the package-lock again, I chose to leave the dependency explicit. The version installed is determined by `@storybook/addon-docs`.

Reply:
Weird, I don't see anything unique about this dependency; it's listed as a dependency of `@storybook/addon-docs` in the package lock.

I would like us to get to the bottom of this but I'd be ok tracking that as follow up tech debt work owned by the Nimble team.